extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/services.js

```diff
@@ -453,6 +453,9 @@ angular.module('PaperUI.services', [ 'PaperUI.services.repositories', 'PaperUI.c
             }
         },
         setConfigDefaults : function(originalConfiguration, groups, sending) {
+            if (!groups) { // no config groups available for this configuration
+                return {};
```

Comment:
hmmm, just a wild guess from looking at the function signature (so I might be completely wrong), but shouldn't it return `originalConfiguration` in this case?

Reply:
good catch. Since originalConfiguration is always {} at this point it technically does not matter. but it looks awful this way.